docs/site/hasOne-relation.md

```diff
@@ -295,3 +295,87 @@ certain properties from the JSON/OpenAPI spec schema built for the `requestBody`
 payload. See its [GitHub
 issue](https://github.com/strongloop/loopback-next/issues/1179) to follow the discussion.
 " %}
+
+## Querying related models
+
+We introduce the concept of `inclusion resolver` in `Has Many Relation` and
```

Comment:
Should we have the same intro as the other relations for consistency: 

```
LoopBack 4 has the concept of an `inclusion resolver` in relations, which helps
to query data through an `include` filter. An inclusion resolver is a function
that can fetch target models for the given list of source model instances.
LoopBack 4 creates a different inclusion resolver for each relation type.
```
?

Reply:
I don't want to repeat (copy) it three times in three different relations 😂 I was thinking that they might check Has Many and BelongsTo already.

Reply:
I thought about that before. But I found that the content in `Relation.html` are high-level stuff. That's why I put them inside of each relations.

Reply:
> IMHO, it might not be the right assumption that users would read the docs for all the relations page. :)

I agree that it might not be the right assumption that users will read the docs for all of them (they might just be interested in `hasOne`).

> Besides, when we have more relations, it might not be a good idea to copy and paste the same paragraph everywhere too. Just my 2 cents.

I think for the meantime since we only have three, it should be fine?

Reply:
+1 to introduce the `InclusionResolver` concept at `relations` level. Each of the specific relations can have its own page. We can have links to refer to the `InclusionResolver`.